czech/druhy_bok/spanek.md

```diff
@@ -1,3 +1,5 @@
 Spíš, ale ve spánku si uvědomuješ sama sebe. Divné. Dokážeš ovlivnit své myšlenky. Co uděláš?
 Spoustu lidí chce létat a ve spánku, pokud jsme si vědomi sami sebe, můžeme. Existuje 
-i odborný termín - anglicky lucid dreaming.
\ No newline at end of file
+i odborný termín - anglicky lucid dreaming.
+
+[Pokusíš se usnout ve snu](sen/senvesnu.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.